Improve External Integrations section (#5709)
wazuh/wazuh-documentation: source/user-manual/manager/manual-integration.rst

  # service wazuh-manager restart

Optional filters
^^^^^^^^^^^^^^^^

> These fields are used by ``Integrator`` to apply certain filters to the alerts being processed and sent to the external platforms. 

Author: Done

The following considerations must be taken into account when the filters are set:
   

>    - The behavior of the ``<group>`` field when multiple values are defined is that of an OR operation. If any of the defined groups matches the alert group, the alert is processed.

Reviewer: Consider adding this suggestion or something similar to it:
```suggestion
   - It is possible to specify multiple group names using the ``<group>`` field a with comma-separated list. If the alert's group matches any of the groups in the list, the alert is sent. Otherwise it is ignored.
```

Author: Done

   - The behavior of the ``<rule_id>`` field when multiple values are defined is that of an OR operation. If any of the defined rule ids matches the alert id, the alert is processed. If the alert does not contain an id or does not match with the defined ones, then it is skipped.

>    - The behavior between the ``<rule_id>`` and ``<group>`` fields when both are defined. If the alert being processed does not match any of the set values or just the values present in one of the fields but not any of the other, it is skipped. If the alert matches at least one of the values of each filter, the alert is processed by the integration.

Reviewer: Consider updating these having in mind the previous suggestion, if required.

Author: Done

  # service wazuh-manager restart

Optional filters
^^^^^^^^^^^^^^^^

> The *optional filters* fields are used by ``Integrator`` to apply certain filters to the alerts being processed to determine which ones should be sent to the external platforms. Only the alerts that meet the filter conditions are sent. If no filters are specified, all alerts are sent.

Reviewer 2: ```suggestion
The `Integrator` daemon uses the `optional filters` fields to determine which alerts should be sent to the external platforms. Only the alerts that meet the filter conditions are sent. If no filters are specified, all alerts are sent.
```

Author: Done

The following considerations must be taken into account when the filters are set:
   
   - It is possible to specify multiple group names using the ``<group>`` field with a comma-separated list. If the alert's group matches any of the groups in the list, the alert is sent. Otherwise, it is ignored.

>    - It is possible to specify multiple rule ids using the ``<rule_id>`` field with a comma-separated list. If the alert's rule id matches any of the ids in the list, the alert is sent. Otherwise, it is ignored.

Reviewer 2: ```suggestion
   - It is possible to specify multiple rule IDs using the ``<rule_id>`` field with a comma-separated list. If the alert's rule ID matches any of the IDs in the list, the alert is sent. Otherwise, it is ignored.
```

Author: Done

>    - It is possible to specify the previously described fields together. If the alert's rule id and group match any of the ids and groups in the lists, the alert is sent. Otherwise, it is ignored.

Reviewer 2: ```suggestion
   - It is possible to specify the previously described fields together. If the alert's rule ID and group match any of the IDs and groups in the lists, the alert is sent. Otherwise, it is ignored.
```

Reviewer 2: I find this explanation a bit ambiguous, I think it is not clear if it is enough that one condition is met or both must be met.

Author: Both must be met in case they are defined together. I will add it to the explanation.
